apps/docs/src/styles.scss

```diff
@@ -282,3 +283,7 @@ body {
         background: radial-gradient(circle, rgba(2, 0, 36, 1) 0%, rgba(0, 0, 0, 0.48) 100%);
     }
 }
+
+.fd-info-label + .fd-info-label {
+    margin-left: 0.3rem;
+}
```

Comment:
What is the reason of changes in this file?

Reply:
@InnaAtanasova had requested that we tag pages as "non-fiori 3" and/or "non-themeable". The example she showed was from the Fundamental-Styles documentation which is using StoryBook. These info-label styles were not present in the Fundamental-NGX documentation, so I needed to include them.